Packs/PAN-OS/ReleaseNotes/1_14_17.md

```diff
@@ -0,0 +1,8 @@
+
+#### Integrations
+##### Palo Alto Networks PAN-OS
+- Added support for *polling* to the ***pan-os-query-logs*** command.
+
+#### Playbooks
+##### New: PAN-OS Query Logs For Indicators V2
```

Comment:
This should be `Panorama Query Logs`, and we can explain that starting in 6.2 the command will poll instead of using the GenericPolling playbook